src/main/java/org/apache/commons/crypto/stream/CipherOutputStream.java

```diff
@@ -388,4 +386,112 @@ protected void freeBuffers() {
     Utils.freeDirectBuffer(inBuffer);
     Utils.freeDirectBuffer(outBuffer);
   }
+
```

Comment:
We may need to consider whether we need a setXXX method to expose it to external change. If not allowed we should not expose the method.